.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules/
 .vdiff/
+src/server/report/data.js
```

Comment:
I don't love that the JSON data goes here, but it was an easy solution to having the files import it and having Rollup "just work". I'm going to be exploring alternatives.